consensus/dummy/consensus.go

```diff
@@ -129,16 +129,19 @@ func verifyHeaderGasFields(config *extras.ChainConfig, header *types.Header, par
 	}
 
 	// Verify header.BaseFee matches the expected value.
-	expectedBaseFee, err := customheader.BaseFee(config, parent, header.Time)
+	headerExtra := customtypes.GetHeaderExtra(header)
+	timeMS := header.Time * 1000
+	if config.IsGranite(header.Time) {
+		timeMS = *headerExtra.TimeMilliseconds
+	}
```

Comment:
Check https://github.com/ava-labs/coreth/pull/1288#issuecomment-3346446701
```suggestion
    headerExtra := customtypes.GetHeaderExtra(header)
    timeMS := customtypes.HeaderTimeMilliseconds(header)
```